server/controllers/MessageController.js

```diff
@@ -169,6 +176,13 @@ class MessageController {
       const inbox = [receiver.rows[0].id, rows[0].id];
       await DB.query(inboxQueryString, inbox);
       const msg = rows[0];
+
+      // deliever messages with twilio
+      client.messages.create({
+        to: '+2348169504447',
```

Comment:
Is there a reason why these numbers are hardcoded?

Reply:
No reason, Would it be okay if I set the number as an environment variable in .env file
Would resolve the typo issue in my README.